src/Serialization/Types.purs

```diff
@@ -81,6 +81,8 @@ module Serialization.Types
   , ProtocolParamUpdate
   , ProtocolVersion
   , ExUnitPrices
+  , GeneralTransactionMetadata
+  , TransactionMetadatum
```

Comment:
This module has multiple exports of the same thing:
```
ExUnitPrices
GeneralTransactionMetadata
ProposedProtocolParameterUpdates
ProtocolParamUpdate
TransactionMetadatum
Update
```
hopefully I haven't missed any.